lib/linguist/languages.yml

```diff
@@ -4485,6 +4485,17 @@ Nix:
   tm_scope: source.nix
   ace_mode: nix
   language_id: 252
+Noir:
+  type: programming
+  aliases:
+  - nargo
+  ace_mode: text
+  extensions:
+  - ".nr"
+  filenames:
+  - Nargo.toml
```

Comment:
You don't want this 😁 This will cause this file to be identified as Noir and then the Noir grammar will be used to try and add syntax highlighting which won't likely work. This is a TOML file so leave it be. If users want it to be identified as Noir and are happy with the syntax highlighting, they can implement an override.

```suggestion
```